Tribler/Core/simpledefs.py

```diff
@@ -133,12 +136,16 @@
 
 # SIGNALS (for internal use)
 SIGNAL_ALLCHANNEL_COMMUNITY = 'signal_allchannel_community'
+SIGNAL_CHANNEL_COMMUNITY = 'signal_channel_community'
```

Comment:
Where are these signals coming from? I think they ended up in this PR by mistake somehow.